install.json

```diff
@@ -1,54 +1,101 @@
 {
     "resources": {
-        "body": [
+        "head": [
+            {
+                "type": "style",
+                "src": "progress-bar.css"
```

Comment:
Moving the CSS to a separate file allows it to not have to be included in the JS, keeping the structure cleaner.